migrate.sh

```diff
@@ -25,40 +22,22 @@ invenio index queue init purge
 #       Just need to create all tables from it.
 invenio db create
 # TODO: add back when pipenv access problem is fixed
-#invenio files location create --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")'/data'
-invenio files location create --default 'default-location' /opt/invenio/var/instance/data
+invenio files location create --default 'default-location'  $(pipenv run invenio shell --no-term-title -c "print(app.instance_path)")'/data'
+# invenio files location create --default 'default-location' /opt/invenio/var/instance/data
 invenio index init --force
 invenio rdm-records custom-fields init
 invenio communities custom-fields init
 
-# Backup FK/PK/unique constraints
-cd scripts
-DB_URI="postgresql://zenodo:zenodo@localhost/zenodo"
-
-psql $DB_URI --tuples-only --quiet -f gen_create_constraints.sql > create_constraints.sql
-psql $DB_URI --tuples-only --quiet -f gen_delete_constraints.sql > delete_constraints.sql
-
-# Drop constraints
-psql $DB_URI -f delete_constraints.sql
-
-# Backup indices
-psql $DB_URI -f backup_indices.sql
-psql $DB_URI -f drop_indices.sql
-
-
 # Run migration
-python -m zenodo_rdm.migrator site/zenodo_rdm/migrator/streams.yaml
-
-# TODO: These should be fixed in the legacy/source
-# Apply various consistency fixes
-
-
-# Restore FK/PK/unique constraints and indices
-psql $DB_URI -f create_constraints.sql
-psql $DB_URI -f restore_indices.sql
+python -m zenodo_rdm.migrator site/zenodo_rdm/migrator/tmp/10k/streams.yaml
 
 # Update ID sequences in DB
-psql $DB_URI -f update_sequences.sql
+# This is developed programatically, not need to run it manually.
```

Comment:
The post_load should have updated all sequences. For documentation purposes is okay, but it looks very much like a command to run. If we want to explain the steps maybe we should start an actual documentation page for it?

Reply:
I found out kind of late that [`_post_load` is never called anywhere](https://github.com/inveniosoftware/invenio-rdm-migrator/blob/bd88485a7cb3d09a69d2b802e0029082dc232188/invenio_rdm_migrator/load/postgresql.py#L134). I then used the query to find the sequences and produced the `update_sequences.sql`.